saleor/graphql/account/types.py

```diff
@@ -89,9 +94,44 @@ class Address(ModelObjectType[models.Address]):
         required=False, description="Address is user's default billing address."
     )
 
+    # temporary copy of meta fields to allow specifying the correct version when
+    # fields were introduced
+    private_metadata = NonNullList(
+        MetadataItem,
+        required=True,
+        description=MetadataDescription.PRIVATE_METADATA
+        + ADDED_IN_310
+        + PREVIEW_FEATURE,
+    )
+    private_metafield = graphene.String(
+        args={"key": graphene.NonNull(graphene.String)},
+        description=(
+            MetadataDescription.PRIVATE_METAFIELD + ADDED_IN_310 + PREVIEW_FEATURE
+        ),
+    )
+    private_metafields = Metadata(
+        args={"keys": NonNullList(graphene.String)},
+        description=(
+            MetadataDescription.PRIVATE_METAFIELDS + ADDED_IN_310 + PREVIEW_FEATURE
+        ),
+    )
+    metadata = NonNullList(
+        MetadataItem,
+        required=True,
+        description=(MetadataDescription.METADATA + ADDED_IN_310 + PREVIEW_FEATURE),
+    )
+    metafield = graphene.String(
+        args={"key": graphene.NonNull(graphene.String)},
+        description=(MetadataDescription.METAFIELD + ADDED_IN_310 + PREVIEW_FEATURE),
+    )
+    metafields = Metadata(
+        args={"keys": NonNullList(graphene.String)},
+        description=(MetadataDescription.METAFIELDS + ADDED_IN_310 + PREVIEW_FEATURE),
+    )
```

Comment:
This is a temporary solution. Will be resolved in https://github.com/saleor/saleor/issues/11702